Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -22,13 +22,15 @@
 
 import sys
 
+from webkitbugspy import Tracker, radar
 from .command import Command
 from .. import local
 
 
 class Conflict(Command):
     name = 'conflict'
     help = "Given the representative Radar ID of a conflicting merge, checkout the branch."
+    INTEGRATION_BRANCH_PREFIX = 'integration'
```

Comment:
cc @briannafan  for thoughts on this, I wonder if this prefix should actually be owned by the repository object in case projects want to vary it. Probably not worth making that change for now.

Reply:
Yeah I'm fine with leaving as is and moving it if it changes.